CMS-Enterprise/mint-app
# [MINT-3221] MTO Table Action Component

# MINT-3221

## Description

* Created `MTOTableActions` component, along with its copy and test
* Add `MTOTableActions` to MTOTable
* Add Solution Library link to SolutionForm
* Remove temporary buttons on Home page




## How to test this change

1.  Navigate to a Model Plan with MTO
2. Ensure that the Table Actions is visible and matches figma
3. Play with buttons and ensure modals still work

#### tablet and smaller screen sizes are still WIP as of 12/4/2024: https://cmsgov.slack.com/archives/C01B9TEC65A/p1733340956122379

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
